release/util/sanity_check_cpp.sh

```diff
@@ -4,4 +4,7 @@
 set -e
 rm -rf ray-template
 ray cpp --generate-bazel-project-template-to ray-template
-pushd ray-template && bash run.sh
+(
+    cd ray-template
+    USE_BAZEL_VERSION=5.4.1 bash run.sh
```

Comment:
maybe add comment on why this needs a different version of bazel

Reply:
I think the wheel verification actually also works with 6.5.0; it does not work with bazel 7.x

added a comment nonetheless.